orchestra/scripts/add_slack_user_ids.py

```diff
@@ -0,0 +1,49 @@
+from annoying.functions import get_object_or_None
+from django.conf import settings
+from slacker import Slacker
+
+from orchestra.models import Worker
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+def add_worker_slack_ids():
+    if not hasattr(settings, 'SLACK_EXPERTS_API_KEY'):
```

Comment:
`getattr(settings, 'SLACK_EXPERTS_API_KEY`, None)` if the key is `''`, the request will fail